tests/language-feature/interfaces/constructor-noncopyable-return.slang

```diff
@@ -0,0 +1,59 @@
+//TEST(compute):COMPARE_COMPUTE(filecheck-buffer=CHECK): -shaderobj -output-using-type
+//TEST_INPUT:ubuffer(data=[0 0 0 0], stride=4):out,name=outputBuffer
+// Note: spirv_by_reference is only supported for passing opaque types, so this test won't produce
```

Comment:
make sure the comments are up to date.

Reply:
I was not able to run this test on vk. Do you any suitable comment for adding to that?

Reply:
That's fine, let's not worry about it for now.